Dockerfile

```diff
@@ -1,33 +1,20 @@
-FROM ubuntu:18.04
```

Comment:
Please do not modify upstream Dockerfiles, but rather create a new one. Please avoid git clone, and consider commiting corpus directly.